lib/packagekit-glib2/pk-offline.c

```diff
@@ -144,10 +173,31 @@ pk_offline_cancel (GCancellable *cancellable, GError **error)
  *
  * Return value: %TRUE for success, else %FALSE and @error set
  *
+ * See: pk_offline_clear_results_with_flags()
+ *
  * Since: 0.9.6
  **/
 gboolean
 pk_offline_clear_results (GCancellable *cancellable, GError **error)
+{
+	return pk_offline_clear_results_with_flags (PK_OFFLINE_FLAGS_NONE, cancellable, error);
+}
+
+/**
+ * pk_offline_clear_results_with_flags:
+ * @flags: bit-or of #PkOfflineFlags
+ * @cancellable: A #GCancellable or %NULL
+ * @error: A #GError or %NULL
+ *
+ * Crears the last offline operation report, which may be success or failure.
```

Comment:
```suggestion
 * Clears the last offline operation report, which may be success or failure.
```

Reply:
Copy&paste thing, the original function has it too ;)